src/main/java/com/incentives/piggyback/user/model/User.java

```diff
@@ -0,0 +1,60 @@
+package com.incentives.piggyback.user.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "userdb", schema = "database1")
```

Comment:
Schema should be userdb and table can be derived out from the entity. This annotation can be removed.